cla_public/apps/checker/tests/test_sessions.py

```diff
@@ -0,0 +1,226 @@
+import unittest
+import uuid
+from flask import json
+from base64 import b64encode
+from flask._compat import text_type
+from werkzeug.http import http_date
+from datetime import datetime
+from cla_public.apps.checker.session import CheckerTaggedJSONSerializer, CheckerSessionObject, MeansTest
```

Comment:
Is importing `MeansTest` from a package that uses it rather than the one that defines (`cla_public.apps.checker.means_test`) intentional

Reply:
No thank you for spotting that out. I will change that